packages/discussions/lib/Discussion.js

```diff
@@ -16,6 +16,8 @@ const upsert = async (id, settings = {}, { pgdb, loaders }, legacyDiscussionId)
     }
   }
 
+  const closed = settings.closed || null
```

Comment:
Currently only works one way: You can close a discussion, but not reopen it. After commit and publish, in my article markdown I see `discussionClosed: false` (correct) but the API still delivers `closed: true` for ownDiscussion.